src/main/java/edition/academy/seventh/repository/BookRepository.java

```diff
@@ -37,7 +42,12 @@ public void addBooksToDataBase(List<Book> books) {
 
   public List<Book> getBooksFromDataBase() {
     entityManager = connectorProvider.getEntityManager();
-    List<Book> bookList = entityManager.createQuery("from Book", Book.class).getResultList();
+
```

Comment:
Krzysiu jak przekonasz mnie dlaczego Criteria Api jest w tym momencie lepsze od zapytania SQL'a, które było zastosowane to przepuszcze Ci PRke.

Moje przeciw:
- mniej czytelne,
- tworzenie nie potrzebnie dodatkowych obiektów,

pamiętaj KISS krzysiu.

Reply:
No więc ehmm CB pozwala wyłapać błędy na etapie kompilacji co do encji :)

Reply:
Tworzenie ewentualnie nadmiarowych obiektów w tym wypadku wydaje mi się nieistotną sprawą